README.md

```diff
@@ -157,10 +168,76 @@ ExPlasma.Transaction.decode(tx_bytes)
 
 ### Generating and Submitting Blocks
 #### Creating a block
+
+```elixir
+utxo = %Utxo{blknum: 1}
```

Comment:
is this documentation really needed? seems like an awful lot to support and maintain... maybe having descriptive test references is enough?